tsconfig.json

```diff
@@ -11,11 +11,12 @@
     "module": "esnext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
-    "noEmit": true,
+    "emitDeclarationOnly": true,
```

Comment:
Previously, `awesome-typescript-loader` was exporting the types. Now we rely on tsc to do it, so we emit those type declarations only, and let babel emit the rest of the transpiled source code